app/views/layouts/activities/email.html.slim

```diff
@@ -0,0 +1,10 @@
+h4
```

Comment:
So far, I just let activity mailer to use this layout. If we need any emails use this layout, it should belong to the layout.